src/main/java/org/apache/maven/plugins/changelog/ChangeLogReport.java

```diff
@@ -283,6 +283,12 @@ public class ChangeLogReport extends AbstractMavenReport {
     @Parameter(defaultValue = "connection", required = true)
     private String connectionType;
 
+    /**
+     * If true, file and file revision information is displayed for each SCM entry.
+     */
+    @Parameter(property = "displayFileAndRevInfo", defaultValue = "true")
+    private boolean displayFileAndRevInfo = true;
```

Comment:
perhaps false by default? What's the current behavior?

Reply:
It was matching the current behavior, so this was backwards compatible. From code perspective I thought it would be nicer to rename it to `hideFileAndRevInfo`. So now the default value is false which feels more logical